cranelift/codegen/src/verifier/mod.rs

```diff
@@ -300,14 +300,14 @@ pub fn verify_context<'a, FOI: Into<FlagsOrIsa<'a>>>(
 struct Verifier<'a> {
     func: &'a Function,
     expected_cfg: ControlFlowGraph,
-    expected_domtree: DominatorTree,
+    expected_domtree: SimpleDominatorTree,
```

Comment:
Is there any reason we have to use the old ("simple") implementation here? Or is the idea that when doing verification, we want to fall back to the more trusted algorithm? (Could we document this reasoning in a comment if so?)

Actually, thinking aloud a bit: I think I'm OK with moving everything over to the new algorithm, including the verifier; it should be enough to have the fuzz-target that checks the algorithm itself against the old algorithm. (If we don't trust that, then we shouldn't use the new algorithm anywhere!)

Reply:
Yes, the idea was to compare this tree to the more trusted algorithm. Changed to the new.